components/ILIAS/Mail/classes/class.ilMimeMail.php

```diff
@@ -247,9 +247,9 @@ protected function build(): void
 
     private function removeHTMLTags(string $maybeHTML): string
     {
-        $maybeHTML = str_ireplace(['<br />', '<br>', '<br/>'], "\n", $maybeHTML);
+        $maybeHTML = str_ireplace(['<br />', '<br>', '<br/>', '</p>'], "\n", $maybeHTML);
```

Comment:
Why do we need to replace the closing `p`, but not the opening tag?

Reply:
You can ignore this change. It's also a relict from a test I did. Will remove this.